pkg/kubelet/kuberuntime/kuberuntime_container_linux.go

```diff
@@ -227,6 +230,11 @@ func (m *kubeGenericRuntimeManager) calculateLinuxResources(cpuRequest, cpuLimit
 			// more info.
 			"memory.oom.group": "1",
 		}
+		if utilfeature.DefaultFeatureGate.Enabled(kubefeatures.InPlacePodVerticalScaling) {
```

Comment:
Could you add a positive and negative test for this feature?  You can set features during unit tests so we should verify that this function behaves as expected with feature gate on/off.

Reply:
Sure will add, Just wanted to know opinion about the overall approach so just differed it.